generators/client/files-react.js

```diff
@@ -126,7 +125,7 @@ const files = {
       path: REACT_DIR,
       templates: [
         'shared/reducers/index.ts',
-        'shared/reducers/action-type.util.ts',
```

Comment:
Cleanup this file at https://github.com/jhipster/generator-jhipster/blob/834581e11706bc20456e6e43aee837b69c5f35c0/generators/cleanup.js#L307